.github/workflows/test.yml

```diff
@@ -77,3 +77,26 @@ jobs:
       - run: CI=false yarn build
         env:
           CI: true
+
+  run-dotrun:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dotrun
+        uses: canonical/install-dotrun@main
+
+      - name: Install dependencies
+        run: /snap/bin/dotrun install
+
+      - name: Build assets
+        run: /snap/bin/dotrun --env CI=false build
+
+      - name: Run dotrun
+        run: |
+          /snap/bin/dotrun &
+
```

Comment:
This looks like it could use some formatting cleanup and it has a trailing `&` or is that necessary for some reason?

Reply:
Yup, this is to make this step run in the background.

Reply:
Ah yes, thanks!